internal/cluster/log.go

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package cluster
+
+import logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+var log = logf.Log.WithName("internal-cluster")
```

Comment:
I realize it's not a *this-PR* issue but we should avoid using the name `log` directly to avoid confusion with the default `go` log.

Reply:
I think at one point a `log` got merged and then since then it was just used as a convention for the project but yeah it could create some confusion. There's a lot of code quality things like that in this repo